DigitalLearningSolutions.Data.Migrations/202103161535_AddResetPasswordTable.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202103161535)]
+    public class AddResetPasswordTable : Migration
+    {
+        public override void Up()
+        {
+            Create.Table("ResetPassword")
+                .WithColumn("ID").AsInt32().NotNullable().PrimaryKey().Identity()
+                .WithColumn("ResetPasswordHash").AsCustom("text").NotNullable()
+                .WithColumn("PasswordResetDateTime").AsCustom("timestamp").NotNullable();
```

Comment:
Ah, I just realised that I wasn't super specific on the ticket, but I think we password reset hash is max 64 characters, so we can create is as string, and given that DLS is only in UK, we should probably use AsDateTime for consistency with other tables/columns.